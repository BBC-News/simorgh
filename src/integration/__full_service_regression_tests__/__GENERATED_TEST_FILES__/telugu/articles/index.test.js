/**
 * @service telugu
 * @pathname /telugu/articles/cq0y4008d4vo
 *
 * THIS TEST SUITE WAS GENERATED BY A BUILD SCRIPT. DO NOT EDIT THIS FILE.
 */

import runA11yTests from '../../../../common/a11y';
import runHeaderTests from '../../../../common/header';
import runFootertests from '../../../../common/footer';
import runSEOtests from '../../../../common/SEO';
import runPerformanceTests from '../../../../common/performance';
runA11yTests();
runHeaderTests();
runFootertests();
runSEOtests();
runPerformanceTests();
it('Image with caption', () => {
  const imageEl = document.querySelector(
    'main figure img, main figure amp-img',
  );
  const imageCaptionEl = document.querySelector('main figure figcaption');
  expect(imageEl).toBeInTheDocument();
  expect(imageEl).toBeTruthy();
  expect(imageEl.getAttribute('src')).toMatchInlineSnapshot(
    `"https://ichef.bbci.co.uk/news/640/cpsdevpb/b931/test/574551d0-fcbd-11e9-9afe-1371928331c5.jpg"`,
  );
  expect(imageCaptionEl).toBeInTheDocument();
  expect(imageCaptionEl.textContent).toBeTruthy();
  expect(imageCaptionEl.textContent).toMatchInlineSnapshot(
    `"ఫొటో క్యాప్షన్, ఈ పరీక్ష చిత్రం, కాపీరైట్ BBC, ఫ్రాన్స్ యొక్క మ్యాప్‌ను చూపిస్తుంది. చిత్రం మొదటి మూడు బ్లాకులలో ఉంది మరియు ఈ శీర్షిక ఉంది."`,
  );
});
