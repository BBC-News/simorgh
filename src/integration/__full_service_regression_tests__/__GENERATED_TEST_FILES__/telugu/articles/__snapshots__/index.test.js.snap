// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Canonical link 1`] = `"http://localhost:7080/telugu/articles/cq0y4008d4vo"`;

exports[`Dir attribute 1`] = `"ltr"`;

exports[`FB admins 1`] = `"100004154058350"`;

exports[`FB app ID 1`] = `"1609039196070050"`;

exports[`Footer Copyright 1`] = `"© 2020  BBC. ఇతర వెబ్‌సైట్లలో సమాచారానికి బీబీసీ బాధ్యత వహించదు. ఇతర వెబ్‌సైట్లకు మా లింకింగ్ విధానం గురించి తెలుసుకోండి."`;

exports[`Header Branding 1`] = `"BBC News, తెలుగు"`;

exports[`Heading level 1 1`] = `"ఈ పరీక్షా వ్యాసం యొక్క శీర్షిక ఇది"`;

exports[`Lang attribute 1`] = `"te"`;

exports[`OG description 1`] = `"డక్‌వర్త్ లూయీస్: క్రికెట్ మ్యాచ్‌లో వర్షం పడితే ఫలితం తేల్చేది వీళ్లే... అసలు ఇది ఎలా పుట్టింది?"`;

exports[`OG image 1`] = `"https://news.files.bbci.co.uk/ws/img/logos/og/telugu.png"`;

exports[`OG image alt 1`] = `"BBC News తెలుగు"`;

exports[`OG locale 1`] = `"te-IN"`;

exports[`OG site name 1`] = `"BBC News తెలుగు"`;

exports[`OG title 1`] = `"డక్‌వర్త్ లూయీస్: క్రికెట్ మ్యాచ్‌లో వర్షం పడితే ఫలితం తేల్చేది వీళ్లే... అసలు ఇది ఎలా పుట్టింది? - BBC News తెలుగు"`;

exports[`OG type 1`] = `"article"`;

exports[`OG url 1`] = `"http://localhost:7080/telugu/articles/cq0y4008d4vo"`;

exports[`Page title 1`] = `"డక్‌వర్త్ లూయీస్: క్రికెట్ మ్యాచ్‌లో వర్షం పడితే ఫలితం తేల్చేది వీళ్లే... అసలు ఇది ఎలా పుట్టింది? - BBC News తెలుగు"`;

exports[`Robots meta tag 1`] = `"noodp,noydir"`;

exports[`Skip to content link 1`] = `"కంటెంట్‌కు వెళ్లండి"`;

exports[`Twitter card 1`] = `"summary_large_image"`;

exports[`Twitter creator 1`] = `"@bbcnewstelugu"`;

exports[`Twitter description 1`] = `"డక్‌వర్త్ లూయీస్: క్రికెట్ మ్యాచ్‌లో వర్షం పడితే ఫలితం తేల్చేది వీళ్లే... అసలు ఇది ఎలా పుట్టింది?"`;

exports[`Twitter image alt 1`] = `"BBC News తెలుగు"`;

exports[`Twitter image src 1`] = `"https://news.files.bbci.co.uk/ws/img/logos/og/telugu.png"`;

exports[`Twitter site 1`] = `"@bbcnewstelugu"`;

exports[`Twitter title 1`] = `"డక్‌వర్త్ లూయీస్: క్రికెట్ మ్యాచ్‌లో వర్షం పడితే ఫలితం తేల్చేది వీళ్లే... అసలు ఇది ఎలా పుట్టింది? - BBC News తెలుగు"`;
